packages/mdc-tab-bar/adapter.ts

```diff
@@ -74,7 +74,7 @@ export interface MDCTabBarAdapter {
    * @param index The index of the tab to activate
    * @param clientRect The client rect of the previously active Tab Indicator
    */
-  activateTabAtIndex(index: number, clientRect: ClientRect): void;
+  activateTabAtIndex(index: number, clientRect?: ClientRect): void;
```

Comment:
the implementation in the component adapter defers to a MDCTab method which already uses an optional parameter. That's why it does not involve other changes.

Reply:
Nice!